### PR TITLE
French tts dailun

### PR DESCRIPTION
**Pull request by lidl2019**
Please ensure you've followed the checklist and provide all the required information *before* requesting a review.

## Required Information

- [x] Reference the issue this is meant to fix or describe it if none exists.
- [x] Full description of changes made.

## Pull Request Checklist

- [x] Coding standards are followed (e.g., [PEP8](https://pep8.org/))
- [x] An entry exists for the container in `docker-compose.yml` or `build.yml`
- [x] The Docker image builds
- [x] The container runs correctly when sent inputs with the changes
- [x] No models or other large files are part of these commits
- [x] A description of the tests already run as part of this PR is present
- [x] CI is set up under `.github/workflows` or is not applicable

### CODE REVIEW

**Comment by lidl2019** [comment]
fixed bugs

File: services/espnet-tts/segment.request.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
```
Comment (JRegimbal) [inline services/espnet-tts/segment.request.json:1]: This file should not be here.

File: services/espnet-tts/segment.response.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
```
Comment (JRegimbal) [inline services/espnet-tts/segment.response.json:1]: This file should not be here.

File: services/espnet-tts/src/espnet_util.py [hunk 3]
```diff
@@ -23,46 +26,78 @@
...
         t2 = time.time()
-        wav = vocoder.inference(c)
+        # wav = vocoder.inference(c)
         t3 = time.time()
     rtf = (t3 - start) / (len(wav) / fs)
     logger.info(f"RTF: {rtf}")
     logger.info(f"Elapsed text2speech: {t2 - start}")
     logger.info(f"Elapsed vocoder: {t3 - t2}")
+
+    # audio = Audio(wav.view(-1).cpu().numpy(), rate=text2speech.fs)
```
Comment (JRegimbal) [inline services/espnet-tts/src/espnet_util.py:102]: Don't add commented code in here!

File: services/espnet-tts/src/espnet_util.py [hunk 4]
```diff
@@ -23,46 +26,78 @@
...
-    minlenratio=0.0,
-    maxlenratio=10.0,
-    use_att_constraint=False,
-    backward_window=1,
-    forward_window=3,
-    # Only for FastSpeech & FastSpeech2
-    speed_control_alpha=1.0,
+
+
+# text2speech = Text2Speech(
```
Comment (JRegimbal) [inline services/espnet-tts/src/espnet_util.py:55]: Delete!

File: services/espnet-tts/Dockerfile [hunk 5]
```diff
@@ -1,7 +1,9 @@
-FROM pytorch/pytorch:1.10.0-cuda11.3-cudnn8-runtime
+FROM pytorch/pytorch:1.12.1-cuda11.3-cudnn8-runtime
 
 RUN apt-get update && apt-get install -y libsndfile1 build-essential && rm -rf /var/lib/apt/lists/*
+RUN apt-get update && apt-get install -y espeak-ng
```
Comment (JRegimbal) [inline services/espnet-tts/Dockerfile:4]: Include in installs above so we clear lists after

File: services/espnet-tts/Dockerfile [hunk 6]
```diff
@@ -10,18 +12,43 @@ USER python
 ENV PATH="/home/python/.local/bin:${PATH}"
 
 RUN pip install --upgrade pip
-COPY /services/espnet-tts/requirements.txt .
+COPY ./requirements.txt .
+# COPY /services/espnet-tts/requirements.txt .
```
Comment (JRegimbal) [inline services/espnet-tts/Dockerfile:16]: You cannot actually use relative paths here for how the build actually works. In order to access the schemas, your working directory for builds **must** be the root of the repository. This is why you needed to copy the schema files below (needlessly).

**Review by JRegimbal — CHANGES_REQUESTED** [review]
This will not work. Your entire context is shifted to the local directory, it **must** be run from the root of the repository. This is why absolute paths are used. Add an entry to `build.yml` for this. Contact me if you have more questions.

File: services/espnet-tts/segment.request.json (hunk 1 above)
Comment (lidl2019) [reply services/espnet-tts/segment.request.json:1]: After I shifted my code to use absolute paths, when I execute "docker build -t tts ." in the "/services/espnet-tts"directory 
The following error raised:
Step 10/25 : COPY /services/espnet-tts/requirements.txt .
COPY failed: file not found in build context or excluded by .dockerignore: stat services/espnet-tts/requirements.txt: file does not exis

**Comment by JRegimbal** [comment]
@lidl2019 Please note that the code as of now does not pass PEP 8 linting. Check the details of the failed GitHub Action for more information.

**Review by JRegimbal — COMMENTED** [review]
Not fully cleaned up (see other comments). I've gone and added the workflow file so you get actions for this and undid your changes in the normal `espnet-tts` directory since that is not supposed to be modified as part of this PR.

**Review by JRegimbal — APPROVED** [review]
Great job, Dailun! Both endpoints work as expected. As you work on your final report, please add a brief summary of how you trained the tacotron model in `services/espnet-tts-fr/README.md`.

**Pull request merged**